arkouda/numeric.py

```diff
@@ -337,12 +346,16 @@ def histogram(pda : pdarray, bins : int=10) -> pdarray:
     # To plot, use only the left edges, and export the histogram to NumPy
     >>> plt.plot(binEdges[:-1], h.to_ndarray())
     """
+    if not isinstance(bins, int):
+        raise TypeError('bins must be an int > 0')
+    if bins < 1:
+        raise ValueError('bins must be 1 or greater')
     if isinstance(pda, pdarray) and isinstance(bins, int):
         repMsg = generic_msg("histogram {} {}".format(pda.name, bins))
         return create_pdarray(repMsg)
     else:
-        raise TypeError("must be pdarray {} and bins must be an int {}".\
-                        format(pda,bins))
+        raise TypeError("pda must be a pdarray, not a {}".\
+                                  format(pda.__class__.__name__,bins))
```

Comment:
two arguments? does `bins` need to be here?

Reply:
jeepers, no, I gotta fix that